packages/turf-nearest-point-on-line/index.ts

```diff
@@ -47,8 +47,17 @@ function nearestPointOnLine<G extends LineString | MultiLineString>(
   pt: Coord,
   options: { units?: Units } = {}
 ): NearestPointOnLine {
-  let closestPt: any = point([Infinity, Infinity], {
+  if (!lines || !pt) {
+    throw new Error("lines and pt are required arguments");
+  }
+
+  let closestPt: Feature<
+    Point,
+    { dist: number; index: number; location: number }
+  > = point([Infinity, Infinity], {
```

Comment:
I think you can replace NearestPointOnLine with this cleaner type definition (where you pass the properties into the generic, rather than extending the Feature), and then this can just become type NearestPointOnLine?

Reply:
Yes, no need for the named type i think!